app/shared/src/__tests__/GestureController.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+describe('GestureController', () => {
+  it('should call NativeModules.RNKiwiGestureController.enableGestures', () => {
+    const GestureController = require('../GestureController').default;
+
```

Comment:
Could this be `Import GestureController from '../GestureController'` instead?

Reply:
It's better to stick with my solution because in the second test suite I'm resetting modules, so still, we will have to require `GestureController`